src/System.ComponentModel.TypeConverter/tests/ContainerTests.cs

```diff
@@ -0,0 +1,765 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information. 
+//
+// System.ComponentModel.Container test cases
+//
+// Authors:
+// 	Gonzalo Paniagua Javier (gonzalo@ximian.com)
+//  Ivan N. Zlatev (contact i-nZ.net)
+
+// Copyright (c) 2006 Novell, Inc. (http://www.novell.com)
+// Copyright (c) 2006 Ivan N. Zlatev
+//
+
+using Xunit;
+using System;
+using System.ComponentModel;
+using System.ComponentModel.Design;
+
+namespace System.ComponentModel.Tests
+{
+	class TestService {
+	}
+	
+	class TestContainer : Container {
+		ServiceContainer _services = new ServiceContainer ();
+		bool allowDuplicateNames;
```

Comment:
Nit: should be annotated as private to conform to guidelines.  Same for other occurrences if there are any.